### ISSUE TITLE
ImageWriter Notice: Undefined index: data

### ISSUE THREAD

**Issue by ihsankurniawan** [mpdf/mpdf#1270]
## I found this bug using mPDF version 8.0.7 when getting string content of code

ErrorException:
Notice: Undefined index: data

  at vendor/mpdf/mpdf/src/Writer/ImageWriter.php:95
  at Mpdf\Writer\ImageWriter->writeImages()
     (vendor/mpdf/mpdf/src/Writer/ResourceWriter.php:117)
  at Mpdf\Writer\ResourceWriter->writeResources()
     (vendor/mpdf/mpdf/src/Mpdf.php:9954)
  at Mpdf\Mpdf->_enddoc()
     (vendor/mpdf/mpdf/src/Mpdf.php:2030)
  at Mpdf\Mpdf->Close()
     (vendor/mpdf/mpdf/src/Mpdf.php:9445)
  at Mpdf\Mpdf->Output('', 'S')
     (src/Controller/EventController.php:91)

so basically $info['data'] isn't set
so i add this code for quick hack, don't know if it will affect a lot but now it's working 

```
<?php
if (empty($info['data'])) {
continue;
}

```

**Comment by finwe** [comment]
Closing as per issue template warning. 

Feel free to open a new issue using the provided issue template with all available issue details, especially a code example to reproduce the issue more easily.
